### PR TITLE
readme: php version requirements update

### PR DESCRIPTION
**Pull request by jkuchar**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -47,11 +47,11 @@ via a web browser. See [security warning](https://nette.org/security-warning).**
 
 Notice: Composer PHP version
 ----------------------------
-This project forces `PHP 5.6` as your PHP version for Composer packages. If you have newer version on production you should change it in `composer.json`.
+This project forces `PHP 7.1` as your PHP version for Composer packages. If you have newer version on production you should change it in `composer.json`.
 ```json
 "config": {
 	"platform": {
-		"php": "7.0"
+		"php": "7.2"
```
Comment (dg) [inline readme.md:56]: 7.1 :)

Comment (jkuchar) [reply readme.md:56]: :-)

File: readme.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ If you like Nette, **[please make a donation now](https://nette.org/donate)**. T
 Requirements
 ------------
 
-PHP 5.6 or higher.
+PHP 7.1 or higher.
```
Comment (dg) [inline readme.md]: This is readme.md for all branches, not only for master branch. So even better is to provide full info about requirement, something like 

`Nette Web-Project for Nette 2.4 requires PHP 5.6. The dev-master version requires PHP 7.1.`

File: readme.md [hunk 3]
```diff
@@ -47,11 +47,11 @@ via a web browser. See [security warning](https://nette.org/security-warning).**
 
 Notice: Composer PHP version
 ----------------------------
-This project forces `PHP 5.6` as your PHP version for Composer packages. If you have newer version on production you should change it in `composer.json`.
+This project forces `PHP 7.1` as your PHP version for Composer packages. If you have newer version on production you should change it in `composer.json`.
 ```json
 "config": {
 	"platform": {
-		"php": "7.0"
+		"php": "7.1"
```
Comment (dg) [inline readme.md]: Sorry, 7.2 was right! 

or `"php": "7.2"   ---> changed to 7.2`

File: readme.md (hunk 2 above)
Comment (jkuchar) [reply readme.md]: fixed

File: readme.md (hunk 3 above)
Comment (jkuchar) [reply readme.md]: I haven't think it though, `7.2` is right. 👍

Comment (jkuchar) [reply readme.md]: fixed

File: readme.md (hunk 1 above)
Comment (jkuchar) [reply readme.md:56]: reverted back to 7.2, see bellow

**Comment by dg** [comment]
Thanks

**Pull request merged**